keps/sig-network/3866-nftables-proxy/README.md

```diff
@@ -1814,18 +1764,14 @@ No
 
 It is not expected to...
 
-### Troubleshooting
-
-<!--
-This section must be completed when targeting beta to a release.
-
-For GA, this section is required: approvers should be able to confirm the
-previous answers based on experience in the field.
+We do not currently have any apples-to-apples comparisons; the
+`nftables` perf job uses more CPU than the corresponding `iptables`
+job, but this is because it doesn't run with `minSyncPeriod: 10s` like
+the `iptables` job does, and so it syncs rule changes more often.
+(However, the that it's able to do so without the cluster falling over
+is a strong indication that it _is_ more efficient.)
```

Comment:
we have some proof that handles large scale more efficiently https://gist.github.com/aojea/f9ca1a51e2afd03621744c95bfdab5b8